tests/publiccloud/run_ltp.pm

```diff
@@ -132,10 +132,15 @@ sub run {
         my @skipped = $whitelist->list_skipped_tests($ltp_env, $ltp_command);
         if (@skipped) {
             $skip_tests = '^(' . join("|", @skipped) . ')$';
+            $skip_tests .= '|' . $ltp_exclude if $ltp_exclude;
         }
+        record_info("Exclude", "Excluding tests: $skip_tests");
+    } elsif ($ltp_exclude) {
+        $skip_tests = $ltp_exclude;
+        record_info("Exclude", "Excluding only 'LTP_COMMAND_EXCLUDE' tests: $skip_tests");
+    } else {
+        record_info("Exclude", "None");
```

Comment:
```suggestion
          else {
              $skip_tests = $ltp_exclude if $ltp_exclude;
        record_info("Exclude", "Excluding only 'LTP_COMMAND_EXCLUDE' tests: $skip_tests");
          } 
          record_info("Excluding tests:", length($skip_tests) ? $skip_tests : "none");
```

Reply:
Not sure whether it is a cleaner solution as instead of simple 3 branch code, we have nested branches.

Reply:
as you prefer

Reply:
And we do not save any branching over either. So I would rather stick to the if-elfsif-else